sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/DoFnTester.java

```diff
@@ -561,7 +562,22 @@ public String getName() {
           return combiner;
         }
       };
-      accumulators.put(name, combiner.createAccumulator());
+
+      // Aggregator instantiation is idempotent
+      if (accumulators.containsKey(name)) {
```

Comment:
This change is *almost* substantive. The thing is that both the `OldDoFn` and its wrapper (or vice versa the `DoFn` and its `OldDoFn` wrapper) initialize all the aggregators. It is simplest to just leave it this way. Open to suggestions.